src/app.md

```diff
@@ -606,25 +606,30 @@ use semver::Version;
 # }
 
 fn run() -> Result<()> {
-    let mut release = Version::parse("0.2.6")?;
-    assert_eq!(release, Version {
-        major: 0,
-        minor: 2,
-        patch: 6,
-        pre: vec!(),
-        build: vec!(),
-    });
+    let version_str = "0.2.6";
+    let mut release = Version::parse(version_str)?;
+
+    assert_eq!(
+        release,
+        Version {
+            major: 0,
+            minor: 2,
+            patch: 6,
+            pre: vec![],
+            build: vec![],
+        }
+    );
 
     release.increment_patch();
-    assert_eq!(release, Version::parse("0.2.7")?);
+    assert_eq!("0.2.7", release.to_string());
```

Comment:
If I were to mark my preference here I would suggest putting the hardcoded string against we are testing on the right.

I probably liked your original version better but it's a matter of taste, so I'll let you chose.